### PR TITLE
Add Trans*Code event page

### PR DESCRIPTION
**Pull request by nanuxbe**

### CODE REVIEW

File: pages/transcode.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+Entrance to the Trans\*Code hack day is included in the [conference
+ticket](/tickets/).
+
+However, if you are just coming to Cardiff for the hack day on Monday,
+[tickets](https://www.eventbrite.com/e/transcode-pycon-uk-2017-tickets-38377166137?aff=utm_source%3Deb_email%26utm_medium%3Demail%26utm_campaign%3Dnew_event_email&utm_term=eventurl_text)
+are free.
+
+We are also able to give away a number of tickets to the whole conference to
+Trans\*Code attendees. Please, see instructions on the EventBrite event
```
Comment (alexwlchan) [inline pages/transcode.md]: nit: this needs a full stop

It also refers to an Eventbrite event (nit: no uppercase “B”), but that’s not mentioned elsewhere in the text – it’s obscured by the URL text. Maybe rewrite the penultimate paragraph to be:

    However, if you are just coming to Cardiff for the hack day on Monday, tickets are
    free. Book your ticket [on Eventbrite](…).

**Review by alexwlchan — CHANGES_REQUESTED** [review]
Thanks!

Because of the way the site is build, every page needs to be linked from somewhere, or in the list of unlinked pages in `pyconuk/views.py`. Without those, the page won’t appear on the site!

Please could you add a link or modify `views.py`, so the new page appears, and then I’ll merge?

**Comment by nanuxbe** [comment]
from email exchange with @inglesp :

> >     I remember last year there was no link to the Trans*Code page from the main PyCon site—could there be one this time? I kept having trouble finding it when (e.g.) using different computers where I didn’t have the direct url to hand.
> 
> Yes!  (Although we're waiting on some design work for this to be possible.)

So I'm adding the link to the unliked pages for now.

**Review by alexwlchan — APPROVED** [review]
Okay, LGTM. Thanks!

**Pull request merged**